pkg/endpoint/policy.go

```diff
@@ -607,14 +607,20 @@ func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (
 	// already running the latest revision, otherwise we have to wait for
 	// the regeneration of the endpoint to complete.
 	policyChanged := l3PolicyChanged || l4PolicyChanged
+	needToRegenerateBPF := optsChanged || policyChanged || e.nextPolicyRevision > e.policyRevision
 
 	e.getLogger().WithFields(logrus.Fields{
-		"policyChanged":       policyChanged,
-		"optsChanged":         optsChanged,
-		"policyRevision.next": e.nextPolicyRevision,
-	}).Debug("Done calculating policy")
-
-	needToRegenerateBPF := optsChanged || policyChanged || e.nextPolicyRevision > e.policyRevision
+		logfields.PolicyRegenerationTime: time.Since(regenerateStart).String(),
+		"policyChanged":                  policyChanged,
+		"optionsChanged":                 optsChanged,
+		"bpfCompilationRequired":         needToRegenerateBPF,
+	}).Info("Completed endpoint policy recalculation")
```

Comment:
Should this be Info or Debug?

Reply:
I purposefully made it an info so we gain visibility into what is happening without having to enable debug. We had very limited visibility into how often policy was recalculated, how long it took and what the implications of the recalculation are.